assets/js/views/Main.vue

```diff
@@ -1,5 +1,6 @@
 <template>
 	<Site
+		v-if="state.startup || state.offline"
```

Comment:
```suggestion
		v-if="state.startup"
```

Wieso hast du hier offline mit reingenommen? Glaube Check auf Startup reicht.

Reply:
Ich hatte Angst, dass wenn die WebSocket Verbindung aufgebaut wird, aber vor der ersten Nachricht wieder zusammenbricht (z.B. durch einen Fehlerhaften Reverse Proxy) dann nichts angezeigt wird.

Reply:
Wenn `offline` gesetzt ist, ist der Hintergrund ohnehin ausgegraut und die Offline Leiste unten ist sichtbar. Wenn wir dann auch noch keine `startup` Nachricht bekommen haben ist es mMn. ok hier nichts anzuzeigen. Würde dann eher schauen, ob wir in der Offline-Nachricht selbst noch bessere Meldungen/Vorschläge für bestimmte Fehlerkonstellationen geben können. Aber anderes Thema.

Der Fall kein Startup und Offline sollte ohnehin eher selten auftauchen. In der Entwicklung passiert mir das, wenn ich das FE starte ohne das die Go-Anwendung läuft. Aber das spielt in der Praxis keine Rolle.